package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@lambdatest/node-fetch-sessions",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
Kindly add the description of the package.

Reply:
Done